lib/experimental/rest-api.php

```diff
@@ -28,6 +28,19 @@ function gutenberg_register_block_editor_settings() {
 }
 add_action( 'rest_api_init', 'gutenberg_register_block_editor_settings' );
 
+
+/**
+ * Registers the template REST API routes.
+ */
+function gutenberg_register_rest_template() {
+	$template_controller = new Gutenberg_REST_Template_Revision_Count('wp_template');
+	$template_controller->register_routes();
+	$template_parts_controller = new Gutenberg_REST_Template_Revision_Count('wp_template_part');
+	$template_parts_controller->register_routes();
+}
+
+add_action( 'rest_api_init', 'gutenberg_register_rest_template' );
```

Comment:
Let's register the endpoints like [this](https://github.com/WordPress/gutenberg/blob/56ec52e296a1381e4cdb2c2b9ef9b03ebe5d7b06/lib/rest-api.php#L105-L122)